### PR TITLE
Use modular toml files in FoundriesFactory tests

### PR DESCRIPTION
**Pull request by mwasilew**

### CODE REVIEW

File: automated/linux/factory-reset/prepare-reset.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ export RESULT_FILE
 TYPE="factory_reset"
 ADDITIONAL_TYPE=""
 LABEL=""
+SOTA_CONFDIR="/etc/sota/conf.d/"
```
Comment (roxell) [inline automated/linux/factory-reset/prepare-reset.sh:13]: Can the trailing '/' be dropped

File: automated/linux/ota-apps/update-apps.sh [hunk 2]
```diff
@@ -11,6 +11,7 @@ TYPE="regular"
 APPNAME=""
 VERSION=""
 DEBUG="false"
+SOTA_CONFDIR="/etc/sota/conf.d/"
```
Comment (roxell) [inline automated/linux/ota-apps/update-apps.sh:14]: Can the trailing '/' be dropped

File: automated/linux/ota-update/download-update.sh [hunk 3]
```diff
@@ -16,6 +16,7 @@ PACMAN_TYPE="ostree+compose_apps"
 U_BOOT_VARIABLE_NAME="foobar"
 U_BOOT_VARIABLE_VALUE="baz"
 DEBUG="false"
+SOTA_CONFDIR="/etc/sota/conf.d/"
```
Comment (roxell) [inline automated/linux/ota-update/download-update.sh:19]: Can the trailing '/' be dropped

File: automated/linux/ota-rollback/download-update.sh [hunk 4]
```diff
@@ -86,12 +87,13 @@ fi
...
+mkdir -p "${SOTA_CONFDIR}"
+cp z-99-aklite-callback.toml "${SOTA_CONFDIR}"
+cp z-99-aklite-disable-reboot.toml "${SOTA_CONFDIR}"
 if [ "${PACMAN_TYPE}" = "ostree" ]; then
-    cp z-99-aklite-callback-ostree.toml /etc/sota/conf.d/
-else
-    cp z-99-aklite-callback.toml /etc/sota/conf.d/
+    cp z-99-ostree.toml "${SOTA_CONFDIR}"
 fi
+
```
Comment (roxell) [inline automated/linux/ota-rollback/download-update.sh:96]: is this needed?
if yes, can add a newline at automated/linux/ota-update/download-update.sh row 99 also?

File: automated/linux/ota-rollback/download-update.sh [hunk 5]
```diff
@@ -15,6 +15,7 @@ export UBOOT_VAR_SET_TOOL
 PACMAN_TYPE="ostree+compose_apps"
 UBOOT_IMAGE_NAME="u-boot.itb"
 DEBUG="false"
+SOTA_CONFDIR="/etc/sota/conf.d/"
```
Comment (roxell) [inline automated/linux/ota-rollback/download-update.sh:18]: Can the trailing '/' be dropped

File: automated/linux/ota-update/static-delta.sh [hunk 6]
```diff
@@ -16,6 +16,7 @@ OTA_THRESHOLD=90
 # than 100 means there should not be enough space for the
 # OTA update
 FILL_SIZE=99
+SOTA_CONFDIR="/etc/sota/conf.d/"
```
Comment (roxell) [inline automated/linux/ota-update/static-delta.sh:19]: Can the trailing '/' be dropped

**Comment by mwasilew** [comment]
I think I fixed all comments.

**Pull request merged**